### PR TITLE
Event Framework and Event Stream Processor updates

### PR DESCRIPTION
**Pull request by vgenev**
* added tracestate tag information in the Event Framework documentation
* added Event Stream Processor service description and SD for trace handling

### CODE REVIEW

File: mojaloop-technical-overview/event-stream-processor/README.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
+# Event Stream Processor Service
```
Comment (mdebarros) [inline mojaloop-technical-overview/event-stream-processor/README.md]: Please can you add a section on the custom mojaloop schema:

1. https://github.com/mojaloop/event-stream-processor/blob/master/config/template-mojaloop.json
2. https://github.com/mojaloop/event-stream-processor#111-create-template

File: mojaloop-technical-overview/event-stream-processor/README.md [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+
+Event Stream Processor consumes event messages from all services side cars as they are created according to the [Event Framework](../event-framework/README.md).
```
Comment (mdebarros) [inline mojaloop-technical-overview/event-stream-processor/README.md]: Suggested update to this paragraph:
```markdown
Event Stream Processor consumes event messages from the `topic-events` topic as a result of messages published by the [event-sidecar](https://github.com/mojaloop/event-sidecar) service. Refer to [Event Framework](../event-framework/README.md) for more information on the overall architecture.
```

File: mojaloop-technical-overview/event-stream-processor/assets/diagrams/sequence/process-flow.plantuml [hunk 3]
```diff
@@ -0,0 +1,277 @@
+/'*****
```
Comment (mdebarros) [inline mojaloop-technical-overview/event-stream-processor/assets/diagrams/sequence/process-flow.plantuml]: I don't see `elasticsearch` in this flow diagram.

Please can you add it, and update the flow accordingly.

File: package.json [hunk 4]
```diff
@@ -9,6 +9,7 @@
     "gitbook-plugin-changelog": "1.0.1",
     "gitbook-plugin-collapsible-chapters": "0.1.8",
     "gitbook-plugin-editlink": "1.0.2",
+    "gitbook-plugin-fontsettings": "^2.0.0",
```
Comment (mdebarros) [inline package.json]: Please remove carrot `^`.

File: mojaloop-technical-overview/event-stream-processor/assets/diagrams/architecture/event-stream-processor-overview.svg [hunk 5]
```diff
@@ -0,0 +1 @@
...ink:href="#K"/></g><g id="I"><use transform="matrix(0.05,0,0,0.05,0,0)" xlink:href="#U"/><use transform="matrix(0.05,0,0,0.05,12,0)" xlink:href="#ad"/><use transform="matrix(0.05,0,0,0.05,15.949999999999998,0)" xlink:href="#X"/><use transform="matrix(0.05,0,0,0.05,25.950000000000003,0)" xlink:href="#T"/><use transform="matrix(0.05,0,0,0.05,34.95,0)" xlink:href="#K"/><use transform="matrix(0.05,0,0,0.05,39.95,0)" xlink:href="#N"/><use transform="matrix(0.05,0,0,0.05,43.900000000000006,0)" xlink:href="#O"/><use transform="matrix(0.05,0,0,0.05,52.900000000000006,0)" xlink:href="#T"/><use transform="matrix(0.05,0,0,0.05,61.900000000000006,0)" xlink:href="#Q"/><use transform="matrix(0.05,0,0,0.05,71.9,0)" xlink:href="#X"/><use transform="matrix(0.05,0,0,0.05,81.9,0)" xlink:href="#W"/><use transform="matrix(0.05,0,0,0.05,87.85000000000001,0)" xlink:href="#O"/><use transform="matrix(0.05,0,0,0.05,96.85000000000001,0)" xlink:href="#ae"/></g><path d="M33 0v-248h34V0H33" id="at"/><g id="J"><use transform="matrix(0.05,0,0,0.05,0,0)" xlink:href="#ah"/><use transform="matrix(0.05,0,0,0.05,12,0)" xlink:href="#Z"/><use transform="matrix(0.05,0,0,0.05,24,0)" xlink:href="#at"/></g></defs></g></svg>
```
[the remark this answers is not in the document]
Comment (vgenev) [reply mojaloop-technical-overview/event-stream-processor/assets/diagrams/architecture/event-stream-processor-overview.svg]: you are right!

File: mojaloop-technical-overview/event-stream-processor/README.md [hunk 6]
```diff
@@ -1,13 +1,40 @@
...
 The Service delivers logs, including audits, and traces to EFK stack with enabled APM plugin. Based on the event message type, the messages are delivered to different indexes in the Elasticsearch.
 
 
+## 1. Pre-requisites
+
+### 1.1 Elasticsearch
+
+Ensure that you have created the `mojatemplate` based on the following config: [template](https://github.com/mojaloop/event-stream-processor/blob/master/config/template-mojaloop.json).
+
+#### 1.1.1 Create Template
```
Comment (mdebarros) [inline mojaloop-technical-overview/event-stream-processor/README.md]: Please don't duplicate stuff. Its already in the event-stream-processor readme. Rather reference it, and point to the readme for more information. People need to go there anyway to get the `template-mojaloop.json` file.

File: mojaloop-technical-overview/event-stream-processor/README.md [hunk 7]
```diff
@@ -1,13 +1,40 @@
 # Event Stream Processor Service
 
-Event Stream Processor consumes event messages from all services side cars as they are created according to the [Event Framework](../event-framework/README.md).
+Event Stream Processor consumes event messages from the `topic-events` topic as a result of messages published by the [event-sidecar](https://github.com/mojaloop/event-sidecar) service. Refer to [Event Framework](../event-framework/README.md) for more information on the overall architecture.
 
 The Service delivers logs, including audits, and traces to EFK stack with enabled APM plugin. Based on the event message type, the messages are delivered to different indexes in the Elasticsearch.
 
 
+## 1. Pre-requisites
```
Comment (mdebarros) [inline mojaloop-technical-overview/event-stream-processor/README.md]: This is a great idea.

I would also mention the APM should be installed along with ElasticSearch. Also, Kibana is recommended for visualization.

**Pull request merged**